app/graphql/types/mutation_type.rb

```
@@ -5,8 +5,9 @@
   class MutationType < GraphQL::Schema::Object
     description 'The mutation root of this schema'
 
-    field :update_photo_title, mutation: Mutations::UpdatePhotoTitle
+    field :update_album_title, mutation: Mutations::UpdateAlbumTitle
     field :update_photo_description, mutation: Mutations::UpdatePhotoDescription
+    field :update_photo_title, mutation: Mutations::UpdatePhotoTitle
```

rubocop (1.69.1)

[convention] Consider moving update_photo_description, update_photo_title to a new type and adding the `update_photo` field instead

(GraphQL/ExtractType)